### DISCUSSION TITLE
"!pvp arena"or "!pvp" command to show rank on highscores

### DISCUSSION THREAD

**Discussion by DSJDAPUFF** [runelite/runelite#17811] [Ideas]
No command for pvp arena :(

We have !lms but pvp arena never got added

Small and nice addition for the people who compete there :)

**Comment by nathan-mccann99** [comment]
+1

**Reply by jeansea6** [reply]
Is it not already in the game? if so i thnk this is a good addition

**Reply by 1pwog** [reply]
Wow I thought I just didn't know what the command was this would be nice to have

**Comment by jeansea6** [comment]
support!

**Comment by OSRSCoding** [comment]
I saw PvP arena as an opportunity to break the top 100 in something niche but sad that I can't post my rank in public chat. Would love to see this become a thing so I can flex a bit.

**Comment by Felanbird** [comment]
Is there an in-game chat message for whichever stat you want tracked? If not then this won't be added, adding commands that can't automatically update just causes confusion for users.

**Reply by DSJDAPUFF** [reply]
What do you mean? Its just a pull from highscores

PvP arena highscores

Just like LMS highscores with "!lms"

**Reply by DSJDAPUFF** [reply]
Hey following up on this, I am still confused by what you mean.

It would be the exact same command as all of the other highscore commands. Infact every single other highscores rank has a command but this one. Tons of the pvp community would appreciate it

**Comment by cooperrholmes** [comment]
support! +1

**Comment by 1pwog** [comment]
Yes please! Would be nice to have, I thought this was already there actually +1 support

**Comment by Grantismo** [comment]
+1

**Comment by nickrishnan** [comment]
+1
